services/purge-cache/src/main.js

```diff
@@ -4,6 +4,7 @@ const path = require('path');
 const _ = require('lodash');
 const config = require('typed-env-config');
 const loader = require('taskcluster-lib-loader');
+const monitor = require('taskcluster-lib-monitor');
```

Comment:
Why are these added in this file and in `services/index` and `services/built-in-workers`?  Perhaps they were added accidentally during the rebase?

Reply:
^^ still needs an answer, I think?